meetings/2022-12-01.md

```diff
@@ -65,23 +68,32 @@ URL: https://github.com/solid/notifications/pull/120
 
 >ACTION: eP to review.
 
+* eP: Reviewed. We can use notification sender/receiver. Client/server terminology not appropriate here.
+* AC: If we change to those terms, we should also update the various diagrams.
+* eP: I can follow up updating the diagrams.
+* SC: So we change to 5 classes of products as in Pavlik's suggestion
+* TT: +1 should be major reduction in confusion
+* AC: +1
+* SC: +1
+* RG: May need to review Terminology
+* SC: +1 after update to classes-of-products
+* eP: +1
 
-### Update Notification Data Model
-URL: https://github.com/solid/notifications/pull/124
-
->* SC: Replaces URL: https://github.com/solid/notifications/pull/60 and resolves.. https://github.com/solid/notifications/issues/123
+ACTION: eP to make suggested change to PR.
 
+ACTION: eP to update diagram if necessary as separate PR
 
-### Remove specific constraint of JSON-LD compacted-document-form
-URL: https://github.com/solid/notifications/pull/122
-
->* SC: Resolves https://github.com/solid/notifications/issues/105
->ACTION: eP to review.
+* RG: Separate PR for diagrams would be better
 
 
-### :bike: shed: Use `object` (closely aligned with AS) instead of `topic` to receive notifications about a resource
-URL: https://github.com/solid/notifications/issues/62
+### Add relationship between `object` and `topic` resources
+URL: https://github.com/solid/notifications/pull/127
 
+* SC: There are different approaches as mentioned in https://github.com/solid/notifications/issues/62#issuecomment-1332663243 but keeping `topic` may be simplest at this time.
+* SC: Closes: [:bike: shed: Use `object` (closely aligned with AS) instead of `topic` to receive notifications about a resource](https://github.com/solid/notifications/issues/62).
+* SC: Aaron and I talked about it yesterday. If we introduce a notion of request activity, to make a request, we should use `object` instead of `topic`. The other option with JSON-LD context, which protects `object`, it would be to much to exclude `object` from protected.
```

Comment:
```suggestion
* SC: Aaron and I talked about it yesterday. If we introduce a notion of request activity, to make a request, we should use `object` instead of `topic`. The other option with JSON-LD context, which protects `object`, it would be too much to exclude `object` from protected.
```